client/src/utils/messages.js

```diff
@@ -0,0 +1,37 @@
+import { defineMessages } from 'react-intl';
```

Comment:
should we add all the translations to this util file (e.g. add a homeMessages for the homepage)

Reply:
yeahh i remember us talking about this before and me being like "eh it's fine" but as i was writing the code i realized we should since we reuse the translations in both mobile and home